io/kqueue.cpp

```diff
@@ -164,11 +187,12 @@ class KQueue : public MasterEventEngine, public CascadingEventEngine {
     }
 
     ssize_t wait_for_events(void** data,
-            size_t count, uint64_t timeout = -1) override {
-        int ret = get_vcpu()->master_event_engine->wait_for_fd_readable(_kq, timeout);
+            size_t count, uint64_t timeout) override {
+        int ret = ::photon::wait_for_fd_readable(_kq, timeout);
         if (ret < 0) return errno == ETIMEDOUT ? 0 : -1;
         if (count > LEN(_events))
             count = LEN(_events);
+        static const struct timespec _tm = {0, 0};
```

Comment:
Why static here, while not in enqueue()?

Reply:
Use on-stack temporary timespec is fine. Changed in latest commit.